query-algebrizer/src/types.rs

```diff
@@ -91,6 +93,17 @@ pub enum FulltextColumn {
     Text,
 }
 
+/// One of the named columns of our transactions table.
+#[derive(PartialEq, Eq, Clone)]
+pub enum TransactionsColumn {
```

Comment:
btw, I think I'm going to need to move these out of `query_algebrizer` for use from pull. *sigh*

Crate proliferation!